tests/test_jjb.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
-from jenkins_job_wrecker.cli import get_xml_root, root_to_yaml
+from jenkins_job_wrecker.cli import get_xml_root, root_to_yaml, get_str_presenter, setup_str_presenter
```

Comment:
`get_str_presenter` is unused... it could even be made "private" by prefixing it via `_`